{
  "comments": [
    "When running Junit Test from Eclipse using the JUnit plugin I get an internal \nerror as soon as the Tests run through without failures or errors. Processing \nof TestCases stops at that point. I have always used the latest stable build \n(currently 20020214) but always got the same error.\n\n\norg.eclipse.swt.SWTException: Failed to execute runnable \n(java.lang.NullPointerException)\n\tat org.eclipse.swt.SWT.error(SWT.java:1887)\n\tat org.eclipse.swt.widgets.Synchronizer.runAsyncMessages\n(Synchronizer.java:96)\n\tat org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:1390)\n\tat org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:1208)\n\tat org.eclipse.ui.internal.Workbench.runEventLoop(Workbench.java:788)\n\tat org.eclipse.ui.internal.Workbench.run(Workbench.java:771)\n\tat \n\norg.eclipse.core.internal.boot.InternalBootLoader.run\n(InternalBootLoader.java:777)\n\tat org.eclipse.core.boot.BootLoader.run(BootLoader.java:319)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke\n(NativeMethodAccessorImpl.java:39)\n\tat \n\nsun.reflect.DelegatingMethodAccessorImpl.invoke\n(DelegatingMethodAccessorImpl.java:25)\n\tat java.lang.reflect.Method.invoke(Method.java:324)\n\tat org.eclipse.core.launcher.Main.basicRun(Main.java:151)\n\tat org.eclipse.core.launcher.Main.run(Main.java:502)\n\tat org.eclipse.core.launcher.Main.main(Main.java:362)\n*** Stack trace of contained exception ***\njava.lang.NullPointerException\n\tat org.eclipse.jdt.internal.junit.ui.FailureRunView.endTest\n(FailureRunView.java:148)\n\tat \n\norg.eclipse.jdt.internal.junit.ui.TestRunnerViewPart$5.run\n(TestRunnerViewPart.java:367)\n\tat org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:29)\n\tat org.eclipse.swt.widgets.Synchronizer.runAsyncMessages\n(Synchronizer.java:93)\n\tat org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:1390)\n\tat org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:1208)\n\tat org.eclipse.ui.internal.Workbench.runEventLoop(Workbench.java:788)\n\tat org.eclipse.ui.internal.Workbench.run(Workbench.java:771)\n\tat \n\norg.eclipse.core.internal.boot.InternalBootLoader.run\n(InternalBootLoader.java:777)\n\tat org.eclipse.core.boot.BootLoader.run(BootLoader.java:319)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke\n(NativeMethodAccessorImpl.java:39)\n\tat \n\nsun.reflect.DelegatingMethodAccessorImpl.invoke\n(DelegatingMethodAccessorImpl.java:25)\n\tat java.lang.reflect.Method.invoke(Method.java:324)\n\tat org.eclipse.core.launcher.Main.basicRun(Main.java:151)\n\tat org.eclipse.core.launcher.Main.run(Main.java:502)\n\tat org.eclipse.core.launcher.Main.main(Main.java:362)",
    "protected code against the null pointer exception.\n\nI wonder what is special about your test suite. Are you using\nTestSetup, TestDecorators, a test that implements ITest directly?\n\n",
    "closing the bug but please provide the information about your TestSuite \nstructure"
  ],
  "commentCreationDates": [
    "2002-03-06T06:55:26+01:00",
    "2002-03-07T09:50:05+01:00",
    "2002-03-08T11:50:25+01:00"
  ],
  "traces": [
    {
      "exceptionType": "org.eclipse.swt.SWTException",
      "message": "Failed to execute runnable  (java.lang.NullPointerException)",
      "elements": [
        {
          "method": "org.eclipse.swt.SWT.error",
          "source": "SWT.java:1887"
        },
        {
          "method": "org.eclipse.swt.widgets.Synchronizer.runAsyncMessages",
          "source": "Synchronizer.java:96"
        },
        {
          "method": "org.eclipse.swt.widgets.Display.runAsyncMessages",
          "source": "Display.java:1390"
        },
        {
          "method": "org.eclipse.swt.widgets.Display.readAndDispatch",
          "source": "Display.java:1208"
        },
        {
          "method": "org.eclipse.ui.internal.Workbench.runEventLoop",
          "source": "Workbench.java:788"
        },
        {
          "method": "org.eclipse.ui.internal.Workbench.run",
          "source": "Workbench.java:771"
        },
        {
          "method": "org.eclipse.core.internal.boot.InternalBootLoader.run",
          "source": "InternalBootLoader.java:777"
        },
        {
          "method": "org.eclipse.core.boot.BootLoader.run",
          "source": "BootLoader.java:319"
        },
        {
          "method": "sun.reflect.NativeMethodAccessorImpl.invoke0",
          "source": "Native Method"
        },
        {
          "method": "sun.reflect.NativeMethodAccessorImpl.invoke",
          "source": "NativeMethodAccessorImpl.java:39"
        },
        {
          "method": "sun.reflect.DelegatingMethodAccessorImpl.invoke",
          "source": "DelegatingMethodAccessorImpl.java:25"
        },
        {
          "method": "java.lang.reflect.Method.invoke",
          "source": "Method.java:324"
        },
        {
          "method": "org.eclipse.core.launcher.Main.basicRun",
          "source": "Main.java:151"
        },
        {
          "method": "org.eclipse.core.launcher.Main.run",
          "source": "Main.java:502"
        },
        {
          "method": "org.eclipse.core.launcher.Main.main",
          "source": "Main.java:362"
        }
      ],
      "number": 0,
      "commentIndex": 0,
      "bugId": "10832",
      "date": "2002-03-06T06:55:26+01:00",
      "product": "JDT",
      "component": "UI",
      "severity": "normal"
    },
    {
      "exceptionType": "java.lang.NullPointerException",
      "elements": [
        {
          "method": "org.eclipse.jdt.internal.junit.ui.FailureRunView.endTest",
          "source": "FailureRunView.java:148"
        },
        {
          "method": "org.eclipse.jdt.internal.junit.ui.TestRunnerViewPart$5.run",
          "source": "TestRunnerViewPart.java:367"
        },
        {
          "method": "org.eclipse.swt.widgets.RunnableLock.run",
          "source": "RunnableLock.java:29"
        },
        {
          "method": "org.eclipse.swt.widgets.Synchronizer.runAsyncMessages",
          "source": "Synchronizer.java:93"
        },
        {
          "method": "org.eclipse.swt.widgets.Display.runAsyncMessages",
          "source": "Display.java:1390"
        },
        {
          "method": "org.eclipse.swt.widgets.Display.readAndDispatch",
          "source": "Display.java:1208"
        },
        {
          "method": "org.eclipse.ui.internal.Workbench.runEventLoop",
          "source": "Workbench.java:788"
        },
        {
          "method": "org.eclipse.ui.internal.Workbench.run",
          "source": "Workbench.java:771"
        },
        {
          "method": "org.eclipse.core.internal.boot.InternalBootLoader.run",
          "source": "InternalBootLoader.java:777"
        },
        {
          "method": "org.eclipse.core.boot.BootLoader.run",
          "source": "BootLoader.java:319"
        },
        {
          "method": "sun.reflect.NativeMethodAccessorImpl.invoke0",
          "source": "Native Method"
        },
        {
          "method": "sun.reflect.NativeMethodAccessorImpl.invoke",
          "source": "NativeMethodAccessorImpl.java:39"
        },
        {
          "method": "sun.reflect.DelegatingMethodAccessorImpl.invoke",
          "source": "DelegatingMethodAccessorImpl.java:25"
        },
        {
          "method": "java.lang.reflect.Method.invoke",
          "source": "Method.java:324"
        },
        {
          "method": "org.eclipse.core.launcher.Main.basicRun",
          "source": "Main.java:151"
        },
        {
          "method": "org.eclipse.core.launcher.Main.run",
          "source": "Main.java:502"
        },
        {
          "method": "org.eclipse.core.launcher.Main.main",
          "source": "Main.java:362"
        }
      ],
      "number": 1,
      "commentIndex": 0,
      "bugId": "10832",
      "date": "2002-03-06T06:55:26+01:00",
      "product": "JDT",
      "component": "UI",
      "severity": "normal"
    }
  ],
  "groupId": "10832",
  "bugId": "10832",
  "date": "2002-03-06T06:55:26+01:00",
  "product": "JDT",
  "component": "UI",
  "severity": "normal"
}